src/studio/src/designer/DataModeling/Converter/Json/Strategy/JsonSchemaGeneralConverter.cs

```diff
@@ -782,6 +791,76 @@ private void HandleComplexType(XmlSchemaComplexType item, WorkList<IJsonSchemaKe
             }
         }
 
+        private void HandleSimpleContentRestriction(XmlSchemaComplexType item, WorkList<IJsonSchemaKeyword> keywords, JsonPointer path)
```

Comment:
Any way to refactor this to make it a bit more readable, get rid of the plumbing parts and be left with the actual flow.

Reply:
not sure refactoring will make it more palatable, I can give it a go, and add some comments to clarify the three things going on here

Reply:
not sure refactoring will make it more palatable, I can give it a go, and add some comments to clarify the three things going on here